parasol/solr/test_solr.py

```diff
@@ -286,9 +299,11 @@ def test_query(self, test_client):
         # not paginated so should be starting at 0
         assert response.start == 0
         # should be the two expected documents
+        # FIXME: these lines have no effect
```

Comment:
Thank you for this catch. Once this is merged, I have this and another test I want to take a look at in any case.

Reply:
It was flagged by my linter! I took a stab at fixing it but the obvious way I thought to fix it resulted in failing tests.